vyper/compiler/__init__.py

```diff
@@ -36,6 +36,7 @@
     "abi": output.build_abi_output,
     "asm": output.build_asm_output,
     "source_map": output.build_source_map_output,
+    "source_map_full": output.build_source_map_output,
```

Comment:
i think the CLI will not differentiate source_map and source_map_full, so we can remove this. if you like, you can put a comment for people who are searching the codebase for the difference between json and cli output.

Reply:
```suggestion
    # sourceMapFull in json output corresponds to source_map in CLI for historical compatibility reasons
```